packages/temporal-middleware/README.md

```diff
@@ -0,0 +1,63 @@
+<!-- sap-cloud-sdk-logo -->
+<!-- This block is inserted by scripts/replace-common-readme.ts. Do not adjust it manually. -->
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
+<!-- sap-cloud-sdk-logo-stop -->
+
+# @sap-cloud-sdk/temporal-middleware
+
+This package contains the [Temporal](https://tc39.es/proposal-temporal/docs/) based middleware for date/time (de-)serialization in the SAP Cloud SDK.
+
+## Installation
+
+```
+$ npm install @sap-cloud-sdk/temporal-middleware
+```
+
+## Usage
+```
+import { temporalDeSerializers_v4 } from '@sap-cloud-sdk/temporal-middleware';
+
+const { businessPartnerApi } = businessPartnerService(temporalDeSerializers_v4);
+businessPartnerApi
+  .entityBuilder()
+  .organizationFoundationDate(
+    Temporal.PlainDateTime.from('1995-12-07T03:24:30')
+  )
+  .build();
+```
+For more detailed overview visit our [Temporal middleware documentation](https://sap.github.io/cloud-sdk/docs/js/features/odata/use-odata-v2-type-safe-client-for-javascript-typescript#using-default-temporal-middleware).
+
+<!-- sap-cloud-sdk-common-readme -->
+<!-- This block is inserted by scripts/replace-common-readme.ts. Do not adjust it manually. -->
+## Support
+
+The recommended way to get in touch with us is to create an issue in our [github repository](https://github.com/SAP/cloud-sdk-js/issues).
```

Comment:
```suggestion
The recommended way to get in touch with us is to create an issue on [GitHub](https://github.com/SAP/cloud-sdk-js/issues).
```

Reply:
Is this the same text as in other packages? I think it should be `GitHub` as it is a name, the rest of my comment is not so important.

Reply:
Ok, just saw this: `<!-- sap-cloud-sdk-common-readme -->`, maybe it would be nice to adjust for all packages, but this is not related to this PR...

Reply:
Will update for other packages in a follow up PR.